tuf-spec.md

```diff
@@ -790,7 +790,10 @@ repo](https://github.com/theupdateframework/specification/issues).
          "version" : VERSION,
          "expires" : EXPIRES,
          "targets" : TARGETS,
-         ("delegations" : DELEGATIONS)
+         ("keys_for_delegations" : {
```

Comment:
IMHO this field should be under `delegations`, but then `delegations` would have to remain a dictionary and we'd have to find another name for the list that contains all the delegations...

Reply:
I think we should keep the keys separate for clarity.

Reply:
Okay, keeping it separate works for me. And what about my suggestion to call it `keys` only?  Because the `keys` field in `root.json` in reality also lists *"keys for delegations"*, i.e. keys to delegate trust to other top-level roles, but is only called `keys`.

Reply:
Calling it `keys` to match `root.json` makes sense.